afe/server.go

```diff
@@ -46,3 +75,11 @@ func (s *afeServer) SubmitTrade(ctx context.Context, req *pb.TradeRequest) (*pb.
 	}
 	return &pb.TradeResponse{}, nil
 }
+
+func newRequestID() string {
+	b := make([]byte, 16)
+	if _, err := rand.Read(b); err != nil {
+		return fmt.Sprintf("req-%d", timestamppb.Now().AsTime().UnixNano())
+	}
+	return hex.EncodeToString(b)
+}
```

Comment:
Może moglibyśmy generować ID tak żeby zagwarantować brak kolizji? Jakoś deterministycznie? Teraz to mało istotne, może powinniśmy otworzyć na to issue?